packages/vscode-extension/src/Logger.ts

```diff
@@ -1,6 +1,10 @@
-import { LogOutputChannel, window } from "vscode";
+import { Output } from "./common/OutputChannel";
+import {
+  createReadableOutputChannel,
+  ReadableLogOutputChannel,
+} from "./project/ReadableLogOutputChannel";
 
-const outputChannel = window.createOutputChannel("Radon IDE", { log: true });
+const outputChannel = createReadableOutputChannel(Output.Ide, true);
```

Comment:
I don't think we need to change this at all, we never access the readable functions of this channel anyway

Reply:
what issues? You can safely revert all the changes in this file. I suppose there were no typing issues before you made the changes